projects/plugins/boost/.eslintrc.js

```diff
@@ -44,6 +44,7 @@ module.exports = {
 		'babel.config.js',
 		'wordpress',
 		'vendor',
+		'tests',
```

Comment:
I am ignoring the tests folder since the `tests/e2e` folder has its own `.eslintrc.js` file extending the standards from the Jetpack e2e tests folder. I thought it might cause some conflict.